lib/load-config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+// ESM seems to break `path` and `fs`, require them first before setting it up..
```

Comment:
`esm` shouldn't impact `require()` calls though. Happy to see the Node.js builtins loaded first, but your comment hints at an `esm` bug. Is this still happening for you?

Reply:
I haven't looked into why, but if I set up ESM first like so:

Reply:
This works for me, Node.js 8.11.0, and `esm@3.0.22`.

Reply:
`esm@3.0.15` was installed, I updated to `3.0.22` and it now works as expected.